resources/user-agents/apps/evolution-generic.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
It may be nice to move these into their own folder: `resources/user-agents/apps/evolution/`